style/style.css

```diff
@@ -1,3 +1,576 @@
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  transform: translateY(5rem);
+  background-color: #414246;
+}
+
+.sectionTitle {
+  color: white;
+  font-size: 1.25rem;
+  font-weight: bold;
+  padding: 2rem 1rem 1rem 1rem;
+}
+
+.partnerContainer {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  margin-top: 1rem;
+}
+
+.partnerLogo {
+  width: 5rem;
+  height: auto;
+  margin: 0.5rem 1rem 0.5rem 1rem;
+}
+
+footer {
+  display: flex;
+  transform: translateY(5rem);
+  padding: 1rem;
+  align-items: center;
+  justify-content: center;
+}
+
+footer div {
+  display: flex;
+}
+
+footer div img {
+  width: 2rem;
+  height: auto;
+}
+
+footer div p {
+  font-size: 1rem;
+}
+
+.rightsCourse {
+  font-size: 0.75rem;
+  padding-left: 1rem;
+}
+
+/* Minimum desktop screen size should be 768px. this will be changed in the next summits when i make the deskopt. */
+@media screen and (min-width: 768px) {
+  #menuButton {
+    display: none;
+  }
+
+  #blurSection {
+    background-color: #f8f8f8;
+  }
+
+  .logBar {
+    display: flex;
+    background-color: #3e3e3e;
+    color: white;
+    font-size: small;
+    width: 100%;
+    justify-content: right;
+    align-items: center;
+    padding-right: 17.5%;
+  }
+
+  .logBarImage {
+    height: 0.8rem;
+    width: auto;
+    cursor: pointer;
+  }
+
+  .logBarItem {
+    padding-left: 2rem;
+    cursor: pointer;
+  }
+
+  nav {
+    display: flex;
+    align-items: center;
+  }
+
+  .navItems {
+    text-decoration: none;
+    padding: 1rem;
+    margin-left: 1rem;
+    color: #716a6a;
+    font-weight: bold;
+  }
+
+  .navItems:hover {
+    background-color: #ff4f38;
+    color: white;
+  }
+
+  .navAbout {
+    color: #ff4f38;
+  }
+
+  .campaign {
+    border: solid 0.3rem red;
+    padding: 0.5rem;
+    margin-left: 3rem;
+  }
+
+  .logoHeader {
+    width: 40%;
+  }
+
+  #home {
+    background-image: url(../assets/mainDesktopBack.svg);
+  }
+
+  #home h2 {
+    font-size: 2.5rem;
+    padding-top: 6.3rem;
+    margin: auto;
+    width: 62%;
+    font-weight: 700;
+  }
+
+  h1 {
+    font-size: 3.9rem;
+    margin: auto;
+    padding-bottom: 0.3rem;
+    width: 62%;
+    font-weight: 700;
+  }
+
+  .brief {
+    width: 62%;
+    font-size: 1.2rem;
+    border: solid 0.6rem white;
+  }
+
+  .date {
+    width: 62%;
+  }
+
+  .place {
+    width: 62%;
+  }
+
+  #mainProgram {
+    transform: translateY(4.5rem);
+  }
+
+  #mainProgram hr {
+    margin: auto;
+    width: 3%;
+  }
+
+  #programContainer {
+    display: flex;
+    width: 70%;
+    margin: auto;
+    padding-top: 2rem;
+    padding-bottom: 3rem;
+  }
+
+  .programCard {
+    display: flex;
+    flex-direction: column;
+    margin: 0.1rem;
+  }
+
+  .programCard:hover {
+    transform: scale(1.1);
+    cursor: pointer;
+  }
+
+  .programImage {
+    margin-bottom: 1rem;
+  }
+
+  .programActivity {
+    padding-bottom: 1rem;
+  }
```

Comment:
- [ ] Please make sure to align the content of the in the main program section , I can see that there is a bit of misalignment. As you are already using the `flex box` I suggest you use `flex-basis` property on the` title` (`p` in `.programActivity`). The `flex-basis` CSS property sets the initial main size of a flex item. It sets the size of the content box unless otherwise set with box-sizing. This way even when the sizes of different titles is different the initially designated size to the element will make the rest of the main cards look like they are aligned.

Your design:

<img width="251" alt="Screenshot 2023-03-17 at 9 34 31 PM" src="https://user-images.githubusercontent.com/44798044/225990006-b8e82e40-ba27-4d60-a133-6cece0a771cf.png">


Behance design :

<img width="469" alt="Screenshot 2023-03-16 at 9 55 16 PM" src="https://user-images.githubusercontent.com/44798044/225726247-252c5a0c-d193-4301-9af4-c4822e560d21.png">